coremltools/test/test_lightgbm_tree_classifier_numeric.py

```diff
@@ -0,0 +1,294 @@
+# Created by Caleb Madrigal
+# Copyright (c) 2018, FireEye Inc. All rights reserved.
+#
+# Use of this source code is governed by a BSD-3-clause license that can be
+# found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+
+import unittest
+import os
+import sys
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..')))
```

Comment:
Why is this necessary?

Reply:
Oops, meant to take that out - it's not necessary.